.github/ISSUE_TEMPLATE/new-repo.md

```diff
@@ -21,18 +21,18 @@ Sponsoring WG:
 
 ## Actions to fulfill
 
-This area is used for the TOC to track the [repo creation
-process](/mechanics/CREATING-A-SANDBOX-REPO.md)
-
+This area is used for the TOC to track the [repo creation process](/mechanics/CREATING-A-SANDBOX-REPO.md).
+The _requestor_ should perform the steps listed below and cross out the checkmarks when done.
+The TOC is involved only in the **TOC Gate** steps.
 
 - [ ] Add this issue to the [TOC project board](https://github.com/orgs/knative/projects/9) for review.
 
-- [ ] Send a PR adding entries for this repo in `/peribolos/knative-sandbox.yaml`
+- [ ] Send a PR adding entries for this repo in `/peribolos/knative-sandbox.yaml`. Please mind the alphabetical order when adding to a list.
```

Comment:
Link to: https://github.com/knative/community/blob/master/peribolos/knative-sandbox.yaml